### DISCUSSION TITLE
Table View: show task hierarchy and add ability to expand / collapse tasks to show / hide subtasks

### DISCUSSION THREAD

**Discussion by EduNetEurope** [Leantime/leantime#2758] [Ideas]
Are there any plans to improve the table view by displaying task hierarchies and adding the ability to expand or collapse tasks to reveal subtasks? Currently, tasks at all levels appear in sequence in both list and table views, without distinction between main tasks and subtasks.

**Comment by marcelfolaron** [comment]
Yes, this is planned for our 4.0 release.

**Reply by developer-melazeta** [reply]
Will there be a way to hide subtasks in the Kanban board?

**Reply by marcelfolaron** [reply]
Actually there is a way already. Using the filter option you can choose the task types you want to see. Just don't select subTasks.
With that said we have a few ideas for subtasks and subtask summaries that may address this issue in particular.
